Mage.Verify/src/main/java/mage/verify/mtgjson/MtgJsonCard.java

```diff
@@ -79,4 +82,32 @@ public String getNameAsASCII() {
     public boolean isUseUnicodeName() {
         return this.asciiName != null && this.name != null && !this.asciiName.equals(this.name);
     }
+
+    /**
+     * @return the Rarity of the card if present in the mtgjson file
+     * null if not present.
+     */
+    public Rarity getRarity() {
```

Comment:
should this be a static method in the Rarity enum e.g. getRarityFromString()? It already has a field matching these strings

Reply:
nope, that's correct -- mtgjson contains own rarity strings, it can be different from xmage names